swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
```

Comment:
👍 

We definitely need to import `Item` so that it can be the parent class of `Clothing`.